contracts/src/v0.8/tests/FunctionsOracleUpgradeableHelper.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
```

Comment:
Is this contract just to test if the migration was successful?

Reply:
Yep, exactly.